src/v/kafka/server/group.h

```diff
@@ -812,6 +813,31 @@ class group final : public ss::enable_lw_shared_from_this<group> {
 
     absl::node_hash_map<model::producer_identity, volatile_tx> _volatile_txs;
     absl::node_hash_map<model::producer_identity, prepared_tx> _prepared_txs;
+
+    struct expiration_info {
+        explicit expiration_info(model::timeout_clock::duration timeout)
+          : timeout(timeout)
+          , last_update(model::timeout_clock::now()) {}
+
+        model::timeout_clock::duration timeout;
+        model::timeout_clock::time_point last_update;
+        bool is_expiration_requested{false};
+
+        model::timeout_clock::time_point deadline() const {
+            return last_update + timeout;
+        }
+
+        bool is_expired(model::timeout_clock::time_point now) const {
```

Comment:
can this call `model::timeout_clock::now()` instead, there would be no need for passing argument ?